<launch> 
    <!-- <include file="$(find openni2_launch)/launch/openni2.launch">
        <arg name="depth_registration" value="true"/>
    </include>  -->
    <rosparam param="use_sim_time">true</rosparam>
    <node pkg="rosbag" type="play" name="player" output="screen" args="--clock --loop /home/miningrox/bagfiles/peopletracking/b_2015-05-21-07-05-57.bag"/> 

    <node name="image_on_3d" pkg="dlib" type="image_on_3d" output="screen">
    	<param name="scale_x" value="1.5"/>
    	<param name="scale_y" value="1"/>
    	<param name="adj_thresh" value="-0.4"/>
        <param name="ip_topic" value="/xtion_left/rgb/image_raw"/>
        <param name="ip_topic_depth" value="/xtion_left/depth/image_raw"/> 
        <param name="op_topic" value="/image_on_3d"/>
        <param name="cam_info" value="/xtion_left/depth/camera_info"/>       
        <param name="svm_path" value="/home/miningrox/Workspaces/test_ws/src/fused_people_detection/dlib/src/examples/build/face_detector.svm"/>  
    </node>
    <node name="tracking_visualization_rviz" pkg="rviz" type="rviz" args="-d /home/miningrox/.rviz/laser_on_image.rviz"/> 
</launch>